package.json

```diff
@@ -4,33 +4,31 @@
   "version": "1.0.0",
   "author": "Tom Coleman <tom@chromatic.com>, Dominic Nguyen <dom@chromatic.com>",
   "dependencies": {
-    "@emotion/server": "^11.4.0",
-    "@storybook/design-system": "7.3.4-canary.347.4551fe8.0",
-    "@storybook/theming": "6.5.0-alpha.63",
-    "cheerio": "^1.0.0-rc.5",
-    "formik": "^1.5.7",
-    "gatsby": "^2.32.3",
-    "gatsby-plugin-emotion": "^5.4.0",
-    "gatsby-plugin-facebook-pixel": "^1.0.3",
-    "gatsby-plugin-google-analytics": "^2.3.0",
-    "gatsby-plugin-layout": "^1.0.15",
-    "gatsby-plugin-react-helmet": "^3.0.12",
-    "gatsby-plugin-sharp": "^2.14.1",
-    "gatsby-plugin-sitemap": "^2.1.0",
+    "@storybook/design-system": "7.8.5",
+    "@storybook/theming": "^6.5.10",
+    "cheerio": "^1.0.0-rc.12",
+    "formik": "^2.2.9",
+    "gatsby": "^4.22.0",
+    "gatsby-plugin-facebook-pixel": "^1.0.8",
```

Comment:
This is a nice to have but, I don't think `gatsby-plugin-facebook-pixel` is used. We certainly don't track those metrics.